### ISSUE TITLE
3.2.0.RC1 Multi-Project build doesn't work

### ISSUE THREAD

**Issue by illerax** [grails/grails-core#10134]
Stacktrace:

* Where:
Build file '/home/user/GRRC1/myapp/build.gradle' line: 36

* What went wrong:
A problem occurred evaluating project ':myapp'.
> No signature of method: static org.gradle.util.ConfigureUtil.configure() is applicable for argument types: (build_f1thc2iuf1z1htewj7trvye6c$_run_closure3$_closure8, org.grails.gradle.plugin.core.PluginDefiner, java.lang.Integer) values: [build_f1thc2iuf1z1htewj7trvye6c$_run_closure3$_closure8@5065bd39, ...]
  Possible solutions: configure(groovy.lang.Closure, java.lang.Object)

Build.gradle:
```
grails {
    exploded = true
    plugins {                     //line:36
        compile project(":myplg")
    }
}
```

**Comment by graemerocher** [comment]
this is probably due to the Gradle 3.0 upgrade, if you use Gradle 2.14 in the meantime you can proceed

**Comment by lhotari** [comment]
The change in Gradle was already in 2.14, so you'd have to use Gradle 2.13 in the meantime.

**Comment by MichaelAstreiko** [comment]
@graemerocher what can be the workaround for grails 3.2.0.RC1 or when there will be 3.2.0.RC2 available?

**Comment by graemerocher** [comment]
the workaround is to use Gradle 2.13, and 3.2 RC2 will be out soon

**Comment by MichaelAstreiko** [comment]
ok, thanks

**Comment by tkvw** [comment]
Will this be included in version 3.1.16?
